torch/fx/_symbolic_trace.py

```diff
@@ -437,6 +437,8 @@ def replace_ph(x):
                                      type_expr=fn_for_analysis.__annotations__.get(name, None))
         arg_names = [next(names_iter) for idx in range(skip_arg_idx, total_args)]
         if isinstance(concrete_args, tuple):
+            if len(arg_names) != len(concrete_args):
+                raise RuntimeError(f"Tracing expected {len(arg_names)} arguments but got {len(concrete_args)} concrete arguments")
```

Comment:
Maybe add a guess of a common reason for this (i.e. "did you unpack where you shouldn't have?").

Reply:
Not sure this is a common reason. I think usually just the number of arguments expected vs. gotten is sufficient to understand what the error might be coming from.